cvat/apps/engine/utils.py

```diff
@@ -395,3 +396,23 @@ def build_annotations_file_name(
         class_name, identifier, 'annotations' if is_annotation_file else 'dataset',
         timestamp, format_name, extension,
     ).lower()
+
+
+def directory_tree(path, max_depth=None, current_depth=0) -> str:
+    if not os.path.exists(path):
+        return f"No such file or directory: {path}"
+
+    output = ""
+    if current_depth > 0:
+        output += "|  " * (current_depth - 1) + "|- "
+
+    output += os.path.basename(path) + "/\n"
+
+    if max_depth is None or current_depth < max_depth:
+        for item in os.listdir(path):
+            full_path = os.path.join(path, item)
+            if os.path.isdir(full_path):
+                output += directory_tree(full_path, max_depth, current_depth + 1)
```

Comment:
Wouldn't using os.walk be simpler and more efficient way here?

Reply:
Alright, Ive tried to use walk from datumaro which is based on os.walk and also supports depth